lib/OpenQA/Setup.pm

```diff
@@ -328,6 +328,60 @@ sub load_plugins {
     OpenQA::Setup::update_config($server->config, @{$server->plugins->namespaces}, "OpenQA::WebAPI::Auth");
 }
 
+sub load_ui_plugins {
```

Comment:
This function name leads to the impression this has something to do with the UI. I guess `load_open_api_plugins` would be better assuming we actually want this OpenAPI-specific.